spec/latest/core/index.html

```diff
@@ -630,6 +630,40 @@ <h3>Discovering a Hydra-powered Web API</h3>
       using HTTP Link Headers is typically not necessary as the concepts
       used in responses from the API will dereference to their
       documentation.</p>
+
+    <p>Another case would involve an <i>ApiDocumentation</i> discover from
+      another Api response. There is no restriction on emitting an HTTP
```

Comment:
I think that the sentence about restricting the `Link` header is redundant. In other words, is there anything in the spec suggesting otherwise?

Actually, I'd got one step further and actually state that a Hydra API SHOULD emit the api doc link every time